test/processor/complex/source/data/config.json

```diff
@@ -1,5 +1,6 @@
 {
   "parameters": {
-    "gzip": false
+    "gzip": false,
+    "inputSizeThreshold": "0B"
```

Comment:
I just remind you that slicer can be used as a CLI but also as a Keboola Component - Processor (docker image).

CLI tests use flags/envs. In the processor tests here there is `config.json`.